### PR TITLE
Feature/bouncy object

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
[(1) Refine bouncyObject powerup](https://trello.com/c/2HeZVfyD/483-1-refine-bouncyobject-powerup)

File: Assets/_PartyParkade/Script/PowerUp/BasePowerUp/Deployable.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public override void Activate(PlayerController playerController)
         // Set the spawn position to be the car's position minus the scaled rotation of the car.
         // This puts the powerup behind the car. 
         Vector3 spawnPosition = carTransform.position - m_throwDistance * rotation;
-        transform.position = new Vector3(spawnPosition.x, GetYCoord(), spawnPosition.z);
+        transform.position = new Vector3(spawnPosition.x, GetYCoord(playerController), spawnPosition.z);
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/PowerUp/BasePowerUp/Deployable.cs:45]: Is this the fix for the floating deploybles?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/BasePowerUp/Deployable.cs:45]: Work around for now

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/BasePowerUp/Deployable.cs:45]: But yeah, it "fixes" the floating deployable

File: Assets/_PartyParkade/Script/PowerUp/BouncyObject.cs [hunk 2]
```diff
@@ -3,9 +3,58 @@
...
+    }
     protected override void HandleTrigger(Collider other) { }
+    
+    protected override void CleanUp()
+    {
+        DisableAllBouncyObjects();
+        base.CleanUp();
+    }
+
+    public override void Activate(PlayerController playerController)
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/PowerUp/BouncyObject.cs:35]: public should be above protected I think

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/BouncyObject.cs]: Fixed

**Pull request merged**